src/Raven.Studio/typescript/models/database/tasks/periodicBackup/amazonSettings.ts

```diff
@@ -94,12 +94,6 @@ abstract class amazonSettings extends backupSettings {
                 onlyIf: () => this.enabled()
             }
         });
-
-        this.credentialsValidationGroup = ko.validatedObservable({
```

Comment:
this is going to test the credentials and the existence of the bucket/vault in the specified region